fuel-merkle/src/common/msb.rs

```diff
@@ -62,11 +62,10 @@ mod test {
         Bytes8,
         Msb,
     };
-    use core::mem::size_of;
 
     #[test]
     fn test_msb_for_bytes_1() {
-        const NUM_BITS: usize = size_of::<Bytes1>() * 8;
```

Comment:
I like the `size_of` approach as it's more obvious to the code reader how this value is constructed. Maybe we should split  this into two constants like `NUM_BYTES` vs `NUM_BITS`?